packages/uniforms-bootstrap4/src/gridClassName.js

```diff
@@ -3,32 +3,59 @@ import filterDOMProps from 'uniforms/filterDOMProps';
 filterDOMProps.register('grid');
 
 function gridClassNamePart (size, value, side) {
-    return `${gridClassNamePartNew(value, side)} ${gridClassNamePartOld(size, value, side)}`;
-}
-
-// bootstrap alpha > 5 now uses flexbox columns
-function gridClassNamePartNew (value, side) {
+    const sizeInfix = size === 'xs' ? '' : `${size}-`;
     return side === 'label'
-        ? `col-${value}`
-        : `col-${(12 - value)}`;
+        ? `col-${sizeInfix}${value}`
+        : `col-${sizeInfix}${(12 - value)}`;
 }
 
-// bootstrap alpha < 6 requires col-size
-function gridClassNamePartOld (size, value, side) {
-    return side === 'label'
-        ? `col-${size}-${value}`
-        : `col-${size}-${(12 - value)}`;
+function compareSizeClass (a, b) {
+    if (a === 'xs') {
+        return -1;
+    }
+
+    if (a === 'sm') {
+        if (b === 'md') {
+            return -1;
+        }
+
+        if (b === 'lg') {
+            return -1;
+        }
+
+        if (b === 'xl') {
+            return -1;
+        }
+    }
+
+    if (a === 'md') {
+        if (b === 'lg') {
+            return -1;
+        }
+
+        if (b === 'xl') {
+            return -1;
+        }
+    }
+
+    if (a === 'lg') {
+        if (b === 'xl') {
+            return -1;
+        }
+    }
+
+    return 1;
```

Comment:
Do these have to be in order? If so, maybe let's have an array with correct order and order them by an index in this array: `sizes.indexOf(a) - sizes.indexOf(b)`?

Reply:
ok.

Maybe?
```
const gridOrder = {
    xs: 1,
    sm: 2,
    md: 3,
    lg: 4,
    xl: 5
};

function compareSizeClass (a, b) {
    return gridOrder[a] - gridOrder[b];
}
```